WASM-TEST,Wasm1 failing

When running the `./everest test` command (fresh build following the [web page](https://project-everest.github.io/)), I get the following error:
```
[...]
⚙ KreMLin auto-detecting tools. Here's what we found:
readlink is: readlink
KreMLin called via: ../_build/src/Kremlin.native
KreMLin home is: /home/cassiersg/everest/kremlin
Wrote WasmSupport.wasm
Wrote Wasm1.wasm
cd .output/Wasm1.wasm && \
  if ! false; then node main.js && echo "\033[01;32m✔\033[00m [WASM-TEST,Wasm1]" || (echo "\033[01;31m✘\033[00m [WASM-TEST,Wasm1]" && false); \
  else ! node main.js && echo "\033[01;32m✔\033[00m [WASM-TEST,Wasm1]" || (echo "\033[01;31m✘\033[00m [WASM-TEST,Wasm1]" && false); fi
node: relocation error: /usr/lib/x86_64-linux-gnu/libnode.so.64: symbol SSL_CTX_set_ciphersuites version OPENSSL_1_1_1 not defined in file libssl.so.1.1 with link time reference
✘ [WASM-TEST,Wasm1]
make: *** [Makefile:196: Wasm1.wasm-test] Error 1
make: Leaving directory '/home/cassiersg/everest/kremlin/test'
```

Version: 131cc2416c
OS: Debian 10

--- a/everest
+++ b/everest
@@ -1060,7 +1060,7 @@ do_test () {
   blue "Running tests (commands shown below)"
   set -x
   # kremlin
-  make -C kremlin/test $parallel_opt everything
+  LD_LIBRARY_PATH= make -C kremlin/test $parallel_opt everything
   # evercrypt
   make -C hacl-star test $parallel_opt
   # qd

